client/components/editable.js

```diff
@@ -66,6 +66,8 @@ function Editable({ edited,
         value={state.content}
         onChange={onContentChange}
         autoExpand={true}
```

Comment:
This _should_ be sufficient, so if it's not then we probably need to go a little deeper.

Reply:
Do you mean the autoexpand? That enables when the user makes a change but it doesn't default to the correct size

Reply:
I do mean the autoexpand. It should be fixed there to make it default to the correct size.